### PR TITLE
Update version for TD/TM JSON schema changes

### PR DESCRIPTION
**Pull request by danielpeintner**
follow-up for https://github.com/w3c/wot-thing-description/pull/1912 which missed to update the version/date.

### CODE REVIEW

**Comment by egekorkan** [comment]
@danielpeintner could you do the change in the TD Schema as well? Tm Schema's date is copied from TD. I know it's a bit weird since the td schema didn't change but not doing that means that next the generator script is called, this date will be overwritten.

File: ontology/wotsec.html [hunk 1]
```diff
@@ -54,8 +54,8 @@ <h2>Introduction</h2>
...
-<section id="name"><h4>name</h4><p>IRI: <code>https://www.w3.org/2019/wot/security#name</code></p><span>Name for query, header, cookie, or uri parameters.</span><table class="def numbered"><tbody><tr><td>Domain includes</td><td><code><a href="#APIKeySecurityScheme">wotsec:APIKeySecurityScheme</a></code><br><code><a href="#BasicSecurityScheme">wotsec:BasicSecurityScheme</a></code><br><code><a href="#BearerSecurityScheme">wotsec:BearerSecurityScheme</a></code><br><code><a href="#DigestSecurityScheme">wotsec:DigestSecurityScheme</a></code><br><code><a href="#PoPSecurityScheme">wotsec:PoPSecurityScheme</a></code></td></tr></tbody></table></section>
+<section id="in"><h4>in</h4><p>IRI: <code>https://www.w3.org/2019/wot/security#in</code></p><span>Specifies the location of security authentication information.</span><table class="def numbered"><tbody><tr><td>Domain includes</td><td><code><a href="#BasicSecurityScheme">wotsec:BasicSecurityScheme</a></code><br><code><a href="#BearerSecurityScheme">wotsec:BearerSecurityScheme</a></code><br><code><a href="#DigestSecurityScheme">wotsec:DigestSecurityScheme</a></code></td></tr></tbody></table></section>
```
Comment (danielpeintner) [inline ontology/wotsec.html:57]:
```suggestion
<section id="in"><h4>in</h4><p>IRI: <code>https://www.w3.org/2019/wot/security#in</code></p><span>Specifies the location of security authentication information.</span><table class="def numbered"><tbody><tr><td>Domain includes</td><td><code><a href="#BasicSecurityScheme">wotsec:BasicSecurityScheme</a></code><br><code><a href="#BearerSecurityScheme">wotsec:BearerSecurityScheme</a></code><br><code><a href="#DigestSecurityScheme">wotsec:DigestSecurityScheme</a></code><br><code><a href="#PoPSecurityScheme">wotsec:PoPSecurityScheme</a></code></td></tr></tbody></table></section>
```

File: ontology/wotsec.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h2>Introduction</h2>
...
+<section id="name"><h4>name</h4><p>IRI: <code>https://www.w3.org/2019/wot/security#name</code></p><span>Name for query, header, cookie, or uri parameters.</span><table class="def numbered"><tbody><tr><td>Domain includes</td><td><code><a href="#APIKeySecurityScheme">wotsec:APIKeySecurityScheme</a></code><br><code><a href="#BasicSecurityScheme">wotsec:BasicSecurityScheme</a></code><br><code><a href="#BearerSecurityScheme">wotsec:BearerSecurityScheme</a></code><br><code><a href="#DigestSecurityScheme">wotsec:DigestSecurityScheme</a></code></td></tr></tbody></table></section>
```
Comment (danielpeintner) [inline ontology/wotsec.html:58]:
```suggestion
<section id="name"><h4>name</h4><p>IRI: <code>https://www.w3.org/2019/wot/security#name</code></p><span>Name for query, header, cookie, or uri parameters.</span><table class="def numbered"><tbody><tr><td>Domain includes</td><td><code><a href="#APIKeySecurityScheme">wotsec:APIKeySecurityScheme</a></code><br><code><a href="#BasicSecurityScheme">wotsec:BasicSecurityScheme</a></code><br><code><a href="#BearerSecurityScheme">wotsec:BearerSecurityScheme</a></code><br><code><a href="#DigestSecurityScheme">wotsec:DigestSecurityScheme</a></code><br><code><a href="#PoPSecurityScheme">wotsec:PoPSecurityScheme</a></code></td></tr></tbody></table></section>
```

**Comment by danielpeintner** [comment]
> @danielpeintner could you do the change in the TD Schema as well?

Done so

**Comment by egekorkan** [comment]
I will be merging this so that I can create the PRs for the resources by tomorrow. It is a very trivial fix.

**Pull request merged**